Sources/Rendering/Misc/SynchronizableRenderWindow/index.js

```diff
@@ -225,9 +225,11 @@ export function vtkSynchronizableRenderWindow(publicAPI, model) {
 
   const addOn = createSyncFunction(publicAPI, model.synchronizerContext);
 
-  Object.keys(addOn).forEach((methodName) => {
-    publicAPI[methodName] = addOn[methodName];
-  });
+  Object.keys(addOn)
+    .filter(methodName => methodName.indexOf('SynchronizedViewId') < 0)
```

Comment:
@jourdain  If we add the methods `getSynchronizedViewId` and `setSynchronizedViewId` to the publicAPI, then in the use case where we don't provide a `synchronizedViewId` in the initial state, we end up recursing over and over again until the call stack gets too large (see line 148 of macro.js).

I'm not sure what the actual solution is, but this fix helped me get things working at least.